scss/elements/_sections.scss

```diff
@@ -1,308 +1,306 @@
-				// can't use :first-child because the markdown parser sometimes leaves empty p tag first :(
-				h3:first-of-type,
-				h4:first-of-type {
-					margin-top: 0;
-				}
-
-				*:last-child {
-					margin-bottom: 0;
-				}
-			}
-
-			.markdown-warning-box {
-				&--container {
-					background-color: $white;
-					padding: ($baseline * 1) 0;
-					position: relative;
-					margin-bottom: 24px;
-				}
-
-				&--icon {
-					font-weight: 700;
-					display: inline-block;
-					position: absolute;
-					top: 50%;
-					left: 0;
-					padding-top: 1px;
-					min-width: 38px;
-					min-height: 38px;
-					margin-top: -19px;
-					font-size: 26px;
-					border: 3px solid #323132;
-					border-radius: 50%;
-					color: #fff;
-					background: #323132;
-					line-height: 29px;
-					text-align: center;
-				}
-
-				&--text {
-					p {
-						font-weight: 700;
-						margin-bottom: ($baseline * 3);
-						padding: 0;
-						font-size: 18px;
-						line-height: 32px;
-					}
-				}
-			}
+    &__title {
+        margin-top: $baseline * 6;
+        margin-bottom: 0;
+    }
+
+    &__subtitle {
+        margin-top: $col;
+        margin-bottom: $baseline;
+    }
+
+    &__content {
+
+        &-number {
+            position: relative;
+            margin-right: ($col);
+            display: block;
+            float: left;
+
+            @include breakpoint(sm) {
+                margin-right: $col;
+                margin-left: 0;
+            }
+
+        }
+
+        &--markdown {
+
+            &:focus {
+                outline: none;
+            }
+
+            @include breakpoint(sm) {
+                padding: 0;
+            }
+
+            @include breakpoint(md) {
+                margin-right: ($col * 3);
+            }
+
+            @include breakpoint(lg) {
+                margin-right: ($col * 2);
+            }
+
+            @include markdown;
+
+            position: relative;
+
+            p {
+                font-size: 18px;
+                font-weight: 400;
+                line-height: 32px;
+                margin: 0 0 ($baseline * 3) 0;
+                padding: 0;
+
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                }
+            }
+
+            ul, ol {
+                margin-bottom: ($baseline * 4);
+                  padding-left: ($col * 2);
+            }
+
+            li {
+                margin: 0 0 $baseline 0;
+                line-height: 32px;
+                font-weight: 400;
+                font-size: 18px;
+                padding: 0;
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                }
+            }
+
+            a {
+                text-decoration: underline;
+                font-size: 18px;
+                font-weight: 400;
+                line-height: 32px;
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                }
+              }
```

Comment:
Prob me that had bad indentation on the original commits, but this brace seems too far indented.